packages/package.go

```diff
@@ -76,7 +88,9 @@ func LoadPackageFromPath(path string) (*Package, error) {
 	if err != nil {
 		return nil, err
 	}
-	pkg := &Package{}
+	pkg := &Package{
+		WorkingDir: "/workdir",
```

Comment:
I wonder if this line is a hint to name the new property as docker does: `WorkDir` instead of `WorkingDir`